src/mame/drivers/mc10.cpp

```diff
@@ -513,7 +513,7 @@ void mc10_state::mc10_video(machine_config &config)
 	RAM(config, m_ram).set_default_size("4K").set_extra_options("8K,20K,32K");
 
 	/* video hardware */
-	SCREEN(config, "screen", SCREEN_TYPE_RASTER).set_raw(9.828_MHz_XTAL / 2, 320, 0, 320, 243, 0, 243);
+	SCREEN(config, "screen", SCREEN_TYPE_RASTER).set_raw(9.828_MHz_XTAL / 2, 320, 0, 320, 243, 0, 240);
```

Comment:
Same screen h/v size as visibile area!? Can't possibly be right, and I'd be surprised if it won't cause all kinds of side effects.
Do we have screen timing parameters for this?

Reply:
Should be common between all MC6847 machines (PAL/NTSC versions).

Reply:
The mc10 was the first (and only) device changed to this interface. I see now how I have no horizontal blanking time. If anyone wants to provide help I could use it.

Reply:
I think I have this much better.